tests/unit_tests/test_ecephys.py

```diff
@@ -155,6 +156,76 @@ def test_trigger_check_electrical_series_reference_electrodes_table(self):
             == "electrodes does not  reference an electrodes table."
         )
 
+    def test_check_electrical_series_dtype_pass(self):
+        electrodes = self.nwbfile.create_electrode_table_region(region=[0, 1, 2, 3, 4], description="all")
+
+        electrical_series = ElectricalSeries(
+            name="elec_series",
+            description="desc",
+            data=np.ones((100, 5), dtype=float),
```

Comment:
```suggestion
            data=np.ones((100, 5), dtype=np.dtype("float64")),
```

Specifying the native Python `float` type defaults to using `np.dtype("float64")`; I'd recommend writing this explicitly instead of letting numpy determine it implicitly